src/App/Components/ArticleSocialSharing/ArticleSocialSharing.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import axios from '../../../configs/axios';
+
+const ArticleSocialSharing = (props) => {
+  const { slug, title } = props;
+  const articleLink = `https://authors-haven-staging.herokuapp.com/articles/${slug}`;
+  const handleShare = (social) => {
+    switch (social) {
+      case 'twitter':
+        axios.post(`api/articles/${slug}/share/twitter`);
+        break;
+      case 'linkedin':
+        axios.post(`api/articles/${slug}/share/linkedin`);
+        break;
+      case 'facebook':
+        // eslint-disable-next-line no-undef
+        FB.ui({
+          method: 'send',
+          link: articleLink
+        });
+        axios.post(`api/articles/${slug}/share/facebook`);
+        break;
+      case 'email':
+        axios.post(`api/articles/${slug}/share/gmail`);
+        break;
+      default:
```

Comment:
what is supposed to be done by the default case?? it might be an action or anything else

Reply:
nothing it is just that ESlint requires it

Reply:
exactly it is required but don't forget to put an instruction inside there

Reply:
@murediane have you worked on @alainburindi's comment above?

Reply:
yes